### PR TITLE
instaloader: add page

### PR DESCRIPTION
**Pull request by 4G3NT**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
4.9.6

### CODE REVIEW

File: pages/linux/instaloader.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# instaloader
+
+> Download pictures, videos, captions, and other metadata from Instagram.
+> Note: You will need to provide Instagram login information if you want high quality media downloads.
```
Comment (kbdharun) [inline pages/linux/instaloader.md:4]:
```suggestion
> Note: You will need to provide Instagram login information if you want high-quality media downloads.
```
I have a minor suggestion here

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/instaloader.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Specify login info and download posts (useful for private profiles):
+
+`instaloader --login {{username}} --password {{password}} {{profile_name}}`
+
+- Skip a target if the first downloaded file has been found (useful for updating Instagram archives):
+
+`instaloader --fast-update {{profile_name}}`
+
+- Also Download stories and IGTV videos (login required):
```
Comment (kbdharun) [inline pages/linux/instaloader.md:31]:
```suggestion
- Download stories and IGTV videos (login required):
```

File: pages/linux/instaloader.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+> Download pictures, videos, captions, and other metadata from Instagram.
+> Note: You will need to provide Instagram login information if you want high quality media downloads.
+> More information: <https://instaloader.github.io>.
+
+- Download a profile:
+
+`instaloader {{profile_name}}`
+
+- Also Download highlights:
```
Comment (kbdharun) [inline pages/linux/instaloader.md:11]:
```suggestion
- Download highlights:
```

**Comment by 4G3NT** [comment]
I forgot to mention one thing: options such as `--stories`, `--igtv` download on top of regular posts, so I don't know if removing the "also" word in the commit is a good idea. That said, I don't think it would matter too much, but being more descriptive is always helpful.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/instaloader.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Download a profile:
+
+`instaloader {{profile_name}}`
+
+- Download highlights:
+
+`instaloader --highlights {{profile_name}}`
+
+- Download posts with geotags (if available) while suppressing any user interaction:
```
Comment (acuteenvy) [inline pages/linux/instaloader.md:15]:
```suggestion
- Download posts with geotags (if available), suppressing any user interaction:
```

**Comment by 4G3NT** [comment]
:+1:

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**